src/Table/Footer.jsx

```diff
@@ -51,64 +51,73 @@ export default function Footer({currentPage, numColumns, numPages, onPageChange}
           className={cssClass.CELL}
           colSpan={numColumns}
         >
-          <Button
-            className={cssClass.BUTTON_SCROLL}
-            disabled={currentPage === 1}
-            onClick={() => selectPage(currentPage - 1)}
-            type="linkPlain"
-            value="Prev"
-          />
-          <div className={cssClass.PAGE_NUMBERS}>
-            {/* Make sure the first page is always visible. */}
-            {visibleRange[0] > 1 && (
-              <Button
-                className={cssClass.BUTTON_PAGE}
-                key={1}
-                onClick={() => selectPage(1)}
-                type="linkPlain"
-                value="1"
-              />
-            )}
-            {/*
-              * Show ellipsis if there's at least one omitted page number between 1 and the start of
-              * the visible rnage.
-              */}
-            {visibleRange[0] > 2 && renderEllipsis()}
-            {visibleRange.map(pageNumber => (
-              <Button
-                className={classnames(
-                  cssClass.BUTTON_PAGE,
-                  pageNumber === currentPage && cssClass.BUTTON_PAGE_SELECTED
-                )}
-                key={pageNumber}
-                onClick={() => selectPage(pageNumber)}
-                type="linkPlain"
-                value={`${pageNumber}`}
-              />
-            ))}
-            {/*
-              * Show ellipsis if there's at least one omitted page number between the end of the
-              * visible range and the last page number.
-              */}
-            {visibleRange[VISIBLE_PAGE_RANGE_SIZE - 1] < numPages - 1 && renderEllipsis()}
-            {/* Make sure the last page is always visible. */}
-            {visibleRange[VISIBLE_PAGE_RANGE_SIZE - 1] < numPages && (
-              <Button
-                className={cssClass.BUTTON_PAGE}
-                key={numPages}
-                onClick={() => selectPage(numPages)}
-                type="linkPlain"
-                value={`${numPages}`}
-              />
+          <div className={cssClass.LOADING_CONTAINER}>
+            {isLoading && (
+              <div className={cssClass.LOADING_OUTER}>
+                <div className={cssClass.LOADING_INNER}>Loading...</div>
```

Comment:
It's to make the styling look right: the outer div has `position: absolute` so that we can put it above the numbers and aligns text to the center. The inner div has `display: inline-block` so that it gets affected by the outer div's centering.